### PR TITLE
added laluka

### PR DESCRIPTION
**Pull request by laluka**
added laluka

### CODE REVIEW

File: streamers.csv [hunk 1]
```diff
@@ -223,6 +223,7 @@ techryptic,
 Th3lazykid,
 theblvkhakr,https://www.youtube.com/@blvkhakr
 thecybermentor,https://www.youtube.com/channel/UC0ArlFuFYMpEewyRBzdLHiw
+TheLaluka,https://twitch.tv/thelaluka
```
Comment (goproslowyo) [inline streamers.csv:226]:
```suggestion
TheLaluka,
```
Just youtube link, the first one links to twitch already.

**Comment by laluka** [comment]
My bad haha, thanks man! :)

**Comment by laluka** [comment]
I also have the youtube one, but it's for replays only, should I publish it here as well?
https://www.youtube.com/@TheLaluka

**Comment by goproslowyo** [comment]
Yea totally :) we’ll show a little YouTube icon next your name.

**Pull request merged**